### PR TITLE
[DONE] Fixing multiple endpoints

### PR DESCRIPTION
**Pull request by SiniWilson**
### Requirements

Fix for multiple endpoint

### CODE REVIEW

File: jazz-build-module/environment-deployment-metadata-loader.groovy [hunk 1]
```diff
@@ -60,6 +60,9 @@ def getEnvironmentLogicalId() {
 		}
 		g_environment_logical_id = environment_logical_id
 	}
+	if(g_service_config['domain'] == "jazz"){
```
Comment (svidhani) [inline jazz-build-module/environment-deployment-metadata-loader.groovy]: Indentation "if (" here and in other files

Comment (SiniWilson) [reply jazz-build-module/environment-deployment-metadata-loader.groovy]: fixed

Comment (SiniWilson) [reply jazz-build-module/environment-deployment-metadata-loader.groovy]: fixed

**Comment by rajeevr2715** [comment]
Looks good.

**Pull request merged**